src/platform/site-wide/sass/modules/_m-layers.scss

```diff
@@ -0,0 +1,42 @@
+// z-index layering
```

Comment:
I'm sort of ambivalent about consolidating all this in a separate file, since I think we should be trying to organize our styles by component/feature and this cuts across that. It's nice to be able to see all the layers in one place, but I wonder if that's much more valuable going forward than just being able to see the list of z-index variables.

I'm curious what @ncksllvn and others think.

Reply:
Pro: it feels like photoshop, you can adjust your layers relaltively quickly. 
Con: the code that sets the layer is far away from the code that sets the rest of the styles.

Relatively easy to pull those values back where they belong as well

Reply:
I think longterm we should consider distributing the values into their modules, but based on the issues we've had with z-indexes conflicting across the site, I feel like this is a safe approach at the moment with it being so explicit and discoverable.

Reply:
Should we add a TODO comment to reorganize this in the future? Should we create VRT tests for this ordering?

Reply:
theyre all trigger by clicking and opening, @cehsu right now, VRT only takes static screen shots